src/baltpeter/Internetmarke/LoadPortokasse.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace baltpeter\Internetmarke;
+
+/**
+ * Helper class to create form
+ * for loading the portokasse
+ */
+class LoadPortokasse
```

Comment:
Unfortunately, there doesn't seem to be any 'official' translation of this process. I personally think charging sounds a little better than loading.